src/components/uploader/css/Uploader.css

```diff
@@ -237,14 +237,23 @@ tr.fl-uploader-file-state-uploaded button:before {
  */
 
 /* Fake add files button wrapper ("Browse files") */
+.fl-force-right{
```

Comment:
Its the name given to the div of that button and center text is used to center the text in that div. Since the button needs to be on the right hand side so name of that div is force-right